datalake/src/main/java/com/sequenceiq/datalake/service/sdx/dr/SdxBackupRestoreService.java

```diff
@@ -87,6 +95,9 @@ public class SdxBackupRestoreService {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(SdxBackupRestoreService.class);
 
+    @Value("${last.backup.day:1}")
+    private int lastBackupDay;
```

Comment:
It's probably better to use the total number of seconds here as the value, to make it easier to configure for a partial day